src/ducks/sendTransaction.ts

```diff
@@ -0,0 +1,103 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import StellarSdk, { Memo, Horizon } from "stellar-sdk";
+import BigNumber from "bignumber.js";
+import { ActionStatus } from "./account";
+
+export const sendTxAction = createAsyncThunk<
```

Comment:
Nice! Do you think it would make it more clear if we created helper functions for every step of the submit process: build, sign, submit?

Reply:
Ya I left it as one because I don't think anywhere else is going to be creating/signing/submitting transactions. For readability I can move the build transaction part, but sign and submit are just one line each so doesn't seem worth it?